### PR TITLE
Update nokogiri

### PR DESCRIPTION
**Pull request by Ana06**
There is a security issue with libxml2 which is used by nokogiri. (CVE-2017-0663).

Supersedes https://github.com/openSUSE/open-build-service/pull/3902 In this Pr the Rubocop fix is added. :bowtie:

### CODE REVIEW

File: src/api/test/unit/schema_test.rb [hunk 1]
```diff
@@ -19,7 +19,6 @@ class SchemaTest < ActiveSupport::TestCase
       if io
         testresult = io.read
         io.close
-        # rubocop:disable Style/NumericPredicate
         assert $? == 0, "#{testfile} does not validate against #{f} -> #{testresult}"
         # rubocop:enable Style/NumericPredicate
       end
```
Comment (eduardoj) [inline src/api/test/unit/schema_test.rb]: This `rubocop:enable` line can also be deleted.

**Comment by DavidKang** [comment]
Apart from my comment, LGTM

File: src/api/test/unit/schema_test.rb (hunk 1 above)
Comment (Ana06) [reply src/api/test/unit/schema_test.rb]: this was automatically corrected by rubocop, but yes :+1:

**Comment by Ana06** [comment]
@eduardoj @DavidKang please re-review it!

File: src/api/test/unit/schema_test.rb (hunk 1 above)
Comment (Ana06) [reply src/api/test/unit/schema_test.rb]: I sent an issue to Rubocop regarding this :wink:

**Comment by coolo** [comment]
note that there is no security problem fixed by this as our nokogiri gem does not bundle libxml but uses the system's libxml and the fix for that cve is out since july

**Pull request merged**